auxiliary/producers/replayer/extractors/mongo.go

```diff
@@ -1,26 +1,27 @@
 package extractors
 
 import (
+	"context"
 	"fmt"
 	"reflect"
 	"time"
-	"context"
-	"github.com/golang/protobuf/proto"
+
+	pb "github.com/bptlab/cepta/models/grpc/replayer"
```

Comment:
pb for protobuf? Why not replayer?

Reply:
Consistency: https://github.com/bptlab/cepta/blob/dev/osiris/usermgmt/server.go#L14